system-tests/project-fixtures/vue/src/style.css

```diff
@@ -0,0 +1,90 @@
+:root {
+  font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 24px;
```

Comment:
Seems like a lot of extra code in this project that isn't necessary for the tests?

Reply:
Will cleanup!

Reply:
Addressed in [ce2c587](https://github.com/cypress-io/cypress/pull/24470/commits/ce2c5876323a89d9600bfc6bb21e539d7d9bded3)